Buildings/Controls/OBC/Utilities/PIDWithAutotuning/Relay/BaseClasses/SamplerWithResetThreshold.mo

```diff
@@ -32,8 +32,7 @@ protected
     annotation (Placement(transformation(origin={0,148}, extent={{-80,-90},{-60,-70}})));
   Buildings.Controls.OBC.CDL.Logical.Or samTri "Sampling trigger"
     annotation (Placement(transformation(extent={{40,-30},{60,-10}})));
-  Buildings.Controls.OBC.CDL.Reals.Greater gre(
-    final h=0)
+  Buildings.Controls.OBC.CDL.Reals.Greater gre(final h=1e-6)
```

Comment:
there is no need to set the `h` to non-zero. As one input is `time` (which is not from the iteratively calculation) and another one is a parameter.

Reply:
Check other comparison blocks. Some of them may not need to have the `h` being non-zero.